errors/middleware-user-agent.md

```diff
@@ -0,0 +1,34 @@
+# Removed req.ua from Middleware API
+
+#### Why This Error Occurred
+
+Your middleware is interacting with `req.ua` and it's being deprecated
```

Comment:
**suggestion:** we could use something more positive than _deprecated_

```suggestion
Your middleware is interacting with `req.ua` and this feature needs to opt-in.
```

First, because deprecated seems to mean "supported but discouraged", while we're completely removing `ua` from the request.

Secondly because I fear we'll scare our beta users, while the mitigation is very easy.